app/controllers/application_controller.rb

```diff
@@ -95,9 +95,9 @@ def filter_params
 
   def oidc
     @oidc ||= OidcClient.new(
-      ENV["ACCOUNT_MANAGER_URL"],
-      ENV["OIDC_CLIENT_ID"],
-      ENV["OIDC_CLIENT_SECRET"],
+      ENV.fetch("ACCOUNT_MANAGER_URL"),
+      ENV.fetch("OIDC_CLIENT_ID"),
```

Comment:
could we squash these back at the end, or are you cool with these being separate?
I'm not sure what I prefer... In some ways this is nicely highlighted and the commit message makes the advantage clear.

See earlier comments [about hte Envar names though](https://github.com/alphagov/finder-frontend/pull/2214/commits/b5c32978b7f5e897802e8c985333df0645ace55e#diff-d5241d488259f32ecbe2f636133e5ddaR47-R48)

Reply:
I'll squash the commit down, since the fetch would be valid right from the start